en/flight_controller/pixhawk4.md

```diff
@@ -98,14 +101,16 @@ make px4_fmu-v5_default
 ```
 
 
-## Debug Port
+## Debug Port {#debug_port}
 
-The system's serial console and SWD interface runs on the **FMU Debug** port, while the I/O console and SWD interface can be accessed via **I/O Debug** port.
+The system's [serial console](https://dev.px4.io/master/en/debug/system_console.html) and SWD interface runs on the **FMU Debug** port, while the I/O console and SWD interface can be accessed via **I/O Debug** port.
 In order to access these ports, the user must remove the *Pixhawk 4* casing.
 
-Both ports have standard serial pinout and can be connected to a standard FTDI cable (3.3V, but it's 5V tolerant) or a [Dronecode probe](https://kb.zubax.com/display/MAINKB/Dronecode+Probe+documentation).
-The pinout uses the standard Dronecode debug connector pinout.
-Please refer to the [wiring](https://dev.px4.io/master/en/debug/system_console.html) page for details of how to wire up this port.
+![Pixhawk 4 Debug Ports](../../assets/flight_controller/pixhawk4/pixhawk4_debug_port.jpg)
+
+The pinout uses the standard [Pixhawk debug connector pinout](https://pixhawk.org/pixhawk-connector-standard/#dronecode_debug).
```

Comment:
@davids5 The Pixhawk 4 states that it uses the Pixhawk debug connector pinout. 
Can you confirm whether or not it also uses JST SM06B connector?

Lastly, the pinout is reverse of my expectation - pin 1 is on the right. Is this consistent with the Pixhawk debug connector definition/expectation? It seems to me we should define this in https://pixhawk.org/pixhawk-connector-standard/#dronecode_debug - add an image of the port with the pins labeled.

Thoughts?




The pin numbering is however reversed to my expectation (pin 1 in on the right).